src/pybind/mgr/volumes/module.py

```diff
@@ -489,7 +489,12 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'periodic_async_work',
             type='bool',
             default=False,
-            desc='Periodically check for async work')
+            desc='Periodically check for async work'),
+        Option(
+            'disable_purge_trash',
+            type='bool',
+            default=False,
+            desc='Setting to "true" will disable purging of subvolumes')
```

Comment:
nit: "Enabling this config disables ..."

Reply:
.. and also note that ongoing purges would backoff.

Reply:
Right. Fixed.